hazelcast/src/main/java/com/hazelcast/instance/impl/BootstrappedInstanceProxy.java

```diff
@@ -16,64 +16,27 @@
 
 package com.hazelcast.instance.impl;
 
-import com.hazelcast.cardinality.CardinalityEstimator;
-import com.hazelcast.client.ClientService;
-import com.hazelcast.cluster.Cluster;
-import com.hazelcast.cluster.Endpoint;
-import com.hazelcast.collection.IList;
-import com.hazelcast.collection.IQueue;
-import com.hazelcast.collection.ISet;
-import com.hazelcast.config.Config;
-import com.hazelcast.core.DistributedObject;
-import com.hazelcast.core.DistributedObjectListener;
 import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.ICacheManager;
-import com.hazelcast.core.IExecutorService;
-import com.hazelcast.core.LifecycleService;
-import com.hazelcast.cp.CPSubsystem;
-import com.hazelcast.crdt.pncounter.PNCounter;
-import com.hazelcast.durableexecutor.DurableExecutorService;
-import com.hazelcast.flakeidgen.FlakeIdGenerator;
 import com.hazelcast.instance.impl.executejar.ExecuteJobParameters;
 import com.hazelcast.jet.Job;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.logging.Logger;
-import com.hazelcast.logging.LoggingService;
-import com.hazelcast.map.IMap;
-import com.hazelcast.multimap.MultiMap;
-import com.hazelcast.partition.PartitionService;
-import com.hazelcast.replicatedmap.ReplicatedMap;
-import com.hazelcast.ringbuffer.Ringbuffer;
-import com.hazelcast.scheduledexecutor.IScheduledExecutorService;
-import com.hazelcast.splitbrainprotection.SplitBrainProtectionService;
-import com.hazelcast.sql.SqlService;
-import com.hazelcast.topic.ITopic;
-import com.hazelcast.transaction.HazelcastXAResource;
-import com.hazelcast.transaction.TransactionContext;
-import com.hazelcast.transaction.TransactionException;
-import com.hazelcast.transaction.TransactionOptions;
-import com.hazelcast.transaction.TransactionalTask;
 
 import javax.annotation.Nonnull;
-import java.util.Collection;
 import java.util.List;
-import java.util.UUID;
-import java.util.concurrent.ConcurrentMap;
 
 // A special HazelcastInstance that has a BootstrappedJetProxy
 @SuppressWarnings({"checkstyle:methodcount"})
-public final class BootstrappedInstanceProxy implements HazelcastInstance {
+public final class BootstrappedInstanceProxy extends AbstractHazelcastInstanceProxy {
```

Comment:
nit: for the warning

```suggestion
public final class BootstrappedInstanceProxy extends AbstractHazelcastInstanceProxy<HazelcastInstance> {
```